src/widget/wtrackmenu.cpp

```diff
@@ -39,21 +39,14 @@ WTrackMenu::WTrackMenu(QWidget* parent,
           m_bPlaylistMenuLoaded(false),
           m_bCrateMenuLoaded(false),
           m_eFilters(flags),
-          m_eIndependentFilters(Filter::IndependentFilters) {
+          m_eTrackModelFilters(Filter::TrackModelFilters) {
     m_pNumSamplers = new ControlProxy(
             "[Master]", "num_samplers", this);
     m_pNumDecks = new ControlProxy(
             "[Master]", "num_decks", this);
     m_pNumPreviewDecks = new ControlProxy(
             "[Master]", "num_preview_decks", this);
 
-    if (!trackModel) {
-        // Check if passed flags are a subset of available subsets
-        VERIFY_OR_DEBUG_ASSERT((m_eIndependentFilters | flags) == m_eIndependentFilters) {
```

Comment:
We should still have this check. Of course the logic needs to be inverted because we have the TrackModel *dependent* flags in the list now.

Reply:
Okay. I will rewrite it to just issue a warning on the console in case TrackModel dependent flags are passed without setting a TrackModel.

Reply:
The `VERIFY_OR_DEBUG_ASSERT` macro does this automatically. You can also use the `--debugAssertBreak` command line argument together with a debugger to catch any debug assertions that are violated, which a simple qWarning or qDebug doesn't do.